Sources/SwiftDocC/Infrastructure/Diagnostics/DiagnosticEngine.swift

```diff
@@ -96,13 +96,22 @@ public final class DiagnosticEngine {
         }
     }
     
+    @available(*, deprecated, message: "Please use flush() instead.")
     public func finalize() {
         workQueue.sync {
             for consumer in self.consumers.sync({ $0.values }) {
                 try? consumer.finalize()
             }
         }
```

Comment:
minor: If you want to shorten this, it can call `flush()`.
```suggestion
        flush()
```